invenio_records_rest/errors.py

```diff
@@ -29,15 +29,169 @@
 from invenio_rest.errors import RESTException
 
 
+#
+# Search
+#
 class MaxResultWindowRESTError(RESTException):
     """Maximum number of results passed."""
 
     code = 400
-    description = "Maximum number of results have been reached."
+
+    def __init__(self, max):
+        """Initialize exception."""
+        self.description = \
+            'Maximum number of results have been reached ({0}).'.format(max)
 
 
+#
+# Query
+#
 class InvalidQueryRESTError(RESTException):
     """Invalid query syntax."""
 
     code = 400
-    description = "Invalid query syntax."
+    description = 'Invalid query syntax.'
+
+
+#
+# CiteProc
+#
+class StyleNotFoundRESTError(RESTException):
+    """No such style."""
+
+    code = 400
+
+    def __init__(self, style):
+        """Initialize exception."""
+        self.description = \
+            '[CiteProc]: Style \'{0}\'could not be found.'.format(style)
```

Comment:
- Can you remove the "[prefixes]" in error messages?
- I would simplify the message to: `Style {0} could not be found.`